scopes/compilation/compiler/workspace-compiler.ts

```diff
@@ -119,11 +123,19 @@ ${this.compileErrors.map(formatError).join('\n')}`);
     }
   }
 
-  private get distDir(): PathOsBasedRelative {
+  private async distDirs(): Promise<PathOsBasedRelative[]> {
```

Comment:
please try to provide backwards compatibility and don't break the api if possible

Reply:
But this is a private function, and I updated the usage. I don't think I can keep the old API for this.

Reply:
we figured this out.